### ISSUE TITLE
Warning about non-fully qualified indentifier under a locally opened module

### ISSUE THREAD

**Issue by vprevosto** [OCamlPro/typerex-lint#184]
In the following code:

```ocaml
module M = struct let x = 1 end

let y = M.(x)
```
`ocp-lint` will complain that `x` is not fully qualified:

```
File "lint_qual.ml", line 3, characters 11-12:
  Warning 1: plugin_typedtree/fully_qualified_identifiers/not-qualified-enough SEVERITY 2
  external identifier "x" is not fully qualified
    (should be "M.x")
```
I'd argue that either no warning should be emitted at all, or that at least it should be possible to disable the emission of the warning in that case, without deactivating the whole category.
